### PR TITLE
Bigger key

### PR DESCRIPTION
**Pull request by YutSean**
Support customize the key length of put worker. In fix mode, the value will be corresponding base64, so that its length will also grow. For WAL blocking problem reproduction.

### CODE REVIEW

File: aries-common/src/main/java/org/apache/aries/common/Constants.java [hunk 1]
```diff
@@ -42,4 +42,6 @@
 
   public static final long ONE_GB = ONE_MB * 1024;
 
+  public static final int TEN = 10;
```
Comment (Reidddddd) [inline aries-common/src/main/java/org/apache/aries/common/Constants.java]: what's the unit of TEN? too casual

**Review by Reidddddd — CHANGES_REQUESTED** [review]
I think you also need to update it in example method

**Comment by Reidddddd** [comment]
> I think you also need to update it in example method

you didn't fixed this one

File: aries-hbase/src/main/java/org/apache/aries/PutWorker.java [hunk 2]
```diff
@@ -64,6 +64,9 @@
   private final Parameter<Enum> value_kind =
       EnumParameter.newBuilder("pw.value_kind", VALUE_KIND.FIXED, VALUE_KIND.class)
                    .setDescription("Value is fixed or random generated").opt();
+  private final Parameter<Integer> key_length =
+      IntParameter.newBuilder("pw.key_length").setDefaultValue(Constants.DEFAULT_KEY_LENGTH_PW).
+          setDescription("How long will be the generated key.").opt();
```
Comment (Reidddddd) [inline aries-hbase/src/main/java/org/apache/aries/PutWorker.java]: The description is not good. "The size of the key."

Comment (YutSean) [reply aries-hbase/src/main/java/org/apache/aries/PutWorker.java]: Fixed in the latest commit.

File: aries-hbase/src/main/java/org/apache/aries/PutWorker.java [hunk 3]
```diff
@@ -185,7 +189,7 @@ public void run() {
       try {
         mutator = connection.getBufferedMutator(param);
         while (running) {
-          String k = ToyUtils.generateRandomString(10);
+          String k = ToyUtils.generateRandomString(key_length.value());
```
Comment (Reidddddd) [inline aries-hbase/src/main/java/org/apache/aries/PutWorker.java:192]: Oh my fault... it is not actually about size of the key... I know why you wrote "How long..."

Comment (Reidddddd) [reply aries-hbase/src/main/java/org/apache/aries/PutWorker.java:192]: Then you should not take my advice.. can change it back, but try to be more descriptive.

**Review by Reidddddd — CHANGES_REQUESTED** [review]

**Pull request merged**